### ISSUE TITLE
Conflict with Simplelog

### ISSUE THREAD

**Issue by maisonaustyn** [clanofartisans/ashita-hush#4]
It appears that simplelog bypasses some of Hushes filters. Specifically the filter for "others defeated enemies"

I haven't tested to see if there are any other conflicts related to this. Would be great to use them together.

**Comment by clanofartisans** [comment]
Good catch! I'll have to do some research on this. Appreciate the feedback!
